packages/react-mutation-mapper/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-mutation-mapper",
-  "version": "0.6.30",
+  "version": "0.7.0-beta.1",
```

Comment:
Publish bot will release the version `0.7.0` (it always releases the next patch, never releases the beta version). Same thing applies to all other `package.json` versions in this PR.